### PR TITLE
Update HTTP and gRPC tutorials

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: docs/getting-started/logs.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
     INFO[2022-06-27 11:50:39] experiment logs from Kubernetes cluster       indented-trace=below ... 
-      time=2022-06-27 15:48:59 level=info msg=task 1: ready : started
-      time=2022-06-27 15:48:59 level=info msg=task 1: ready : completed
-      time=2022-06-27 15:48:59 level=info msg=task 2: ready : started
-      time=2022-06-27 15:48:59 level=info msg=task 2: ready : completed
-      time=2022-06-27 15:48:59 level=info msg=task 3: http : started
-      time=2022-06-27 15:49:11 level=info msg=task 3: http : completed
-      time=2022-06-27 15:49:11 level=info msg=task 4: assess : started
-      time=2022-06-27 15:49:11 level=info msg=task 4: assess : completed
+      time=2022-06-27 15:48:59 level=info msg=task 1: ready: started
```
Comment (Alan-Cha) [inline docs/getting-started/logs.md:5]: This was a change I made previously but I see it was never updated in the docs.

**Review by kalantar — DISMISSED** [review]
This looks good. Approving; my comments are nice to have but not necessary.

File: docs/getting-started/your-first-experiment.md [hunk 2]
```diff
@@ -28,28 +28,54 @@ Run your first [Iter8 experiment](concepts.md#iter8-experiment) by load testing
...
---set "tasks={ready,http,assess}" \
---set ready.deploy=httpbin \
---set ready.service=httpbin \
---set ready.timeout=60s \
---set http.url=http://httpbin.default/get \
---set assess.SLOs.upper.http/latency-mean=50 \
---set assess.SLOs.upper.http/error-count=0 \
---set runner=job
-```
+=== "Get example"
```
Comment (sriumcp) [inline docs/getting-started/your-first-experiment.md:31]: GET

File: docs/getting-started/your-first-experiment.md [hunk 3]
```diff
@@ -28,28 +28,54 @@ Run your first [Iter8 experiment](concepts.md#iter8-experiment) by load testing
...
+        
+        The [ready](../user-guide/tasks/ready.md) task checks if the `httpbin` deployment exists and is available, and the `httpbin` service exists. 
+        
+        The [http](../user-guide/tasks/http.md) task sends requests to the cluster-local HTTP service whose URL is `http://httpbin.default/get`, and collects [Iter8's built-in HTTP load test metrics](../user-guide/tasks/http.md#metrics). 
+        
+        The [assess](../user-guide/tasks/assess.md) task verifies if the app satisfies the specified SLOs: i) the mean latency of the service does not exceed 50 msec, and ii) there are no errors (4xx or 5xx response codes) in the responses. 
+        
+        This is a [single-loop](concepts.md#iter8-experiment) [Kubernetes experiment](concepts.md#kubernetes-experiments) where all the previously mentioned tasks will run once and the experiment will finish. Hence, its [runner](concepts.md#runners) value is set to `job`.
+
+=== "Post example"
```
Comment (sriumcp) [inline docs/getting-started/your-first-experiment.md:55]: POST

File: docs/tutorials/load-test-grpc.md [hunk 4]
```diff
@@ -16,44 +16,129 @@ Load test a Kubernetes gRPC service and validate its [service-level objectives (
...
+    --set grpc.call="routeguide.RouteGuide.ListFeatures" \
+    --set grpc.protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/route_guide/routeguide/route_guide.proto" \
+    --set grpc.dataURL="https://raw.githubusercontent.com/iter8-tools/docs/main/samples/grpc-payload/server.json" \
+    --set assess.SLOs.upper.grpc/error-rate=0 \
+    --set assess.SLOs.upper.grpc/latency/mean=200 \
+    --set assess.SLOs.upper.grpc/latency/p'97\.5'=800 \
+    --set runner=job
+    ```
+
+    ??? note "About this experiment"
```
Comment (sriumcp) [inline docs/tutorials/load-test-grpc.md:72]: Let us combine the "About this experiment" notes across the 4 tabs for gRPC and 2 tabs for HTTP.

File: docs/tutorials/load-test-grpc.md [hunk 5]
```diff
@@ -16,44 +16,129 @@ Load test a Kubernetes gRPC service and validate its [service-level objectives (
     1. Try [your first experiment](../getting-started/your-first-experiment.md). Understand the main [concepts](../getting-started/concepts.md) behind Iter8 experiments.
     2. Deploy the sample gRPC service in the Kubernetes cluster.
     ```shell
-    kubectl create deploy hello --image=docker.io/grpc/java-example-hostname:latest --port=50051
-    kubectl expose deploy hello --port=50051
+    kubectl create deployment routeguide  --image=golang --port=50051 -- bash -c "git clone -b v1.52.0 --depth 1 https://github.com/grpc/grpc-go; cd grpc-go/examples/route_guide; go run server/server.go"
```
Comment (sriumcp) [inline docs/tutorials/load-test-grpc.md:19]: Perhaps `\` new line break before `-- bash`?

**Review by sriumcp — CHANGES_REQUESTED** [review]

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**